content/post/text-analysis-of-the-10th-republican-presidential-candidate-debate-using-r-and-the-quanteda-package.md

```diff
@@ -0,0 +1,467 @@
+    ## [6] "An enthusiastic crowd is on hand here in the beautiful opera house at the Moore School of Music. Texas is the biggest prize next Tuesday, Super Tuesday, when 11 states vote, a day that will go a long way towards deciding who wins the Republican nomination."
+    tail(txt, 10)
+    ##  [1] "Nobody knows politicians better than I do. They're all talk, they're no action, nothing gets done. I've watched it for years. Take a look at what's happening to our country."                                                                                                                                                                                                                                           
+    ##  [2] "All of the things that I've been talking about, whether it's trade, whether it's building up our depleted military, whether it's taking care of our vets, whether it's getting rid of Common Core, which is a disaster, or knocking out Obamacare and coming up with something so much better, I will get it done. Politicians will never, ever get it done. And we will make America great again. Thank you. [applause]"
+    ##  [3] "BLITZER: Mr. Trump, thank you."                                                                                                                                                                                                                                                                                                                                                                                          
+    ##  [4] "And thanks to each of the candidates, on behalf of everyone here at CNN and Telemundo. We also want to thank the Republican National Committee and the University of Houston. My thanks also to Hugh Hewitt, Maria Celeste, and Dana Bash."                                                                                                                                                                              
+    ##  [5] "Super Tuesday is only five days away."                                                                                                                                                                                                                                                                                                                                                                                   
+    ##  [6] "Presidential Candidate Debates, Republican Candidates Debate in Houston, Texas Online by Gerhard Peters and John T. Woolley, The American Presidency Project https://www.presidency.ucsb.edu/node/312591"                                                                                                                                                                                                                
```

Comment:
This part needs to be removed from the corpus as it is not part of the debate.

Reply:
True but since it's tagged as "Blitzer" it gets removed. I could add a note to this effect.

Reply:
Added in fce6b0d.